.github/workflows/codeql-analysis.yml

```diff
@@ -42,7 +42,7 @@ jobs:
 
     # Initializes the CodeQL tools for scanning.
     - name: Initialize CodeQL
-      uses: github/codeql-action/init@v1
+      uses: github/codeql-action/init@v2
```

Comment:
We can get dependabot to keep these up to date.

I added this to `django-bleach` recently;

https://github.com/marksweb/django-bleach/blob/main/.github/dependabot.yml

Reply:
I've copied the yml-file.